src/frontend/Dockerfile

```diff
@@ -7,7 +7,6 @@ ENV NODE_ENV=development
 ENV BUILD_PATH=build
 RUN npm i --verbose
 RUN apt-get update && apt-get install make
-RUN npm install -g serve
```

Comment:
was this causing errors?

Reply:
Ah no, we had to replace the usage of `serve` with `Express`, so we can add custom security headers 😄 !